app/assets/javascripts/discourse/app/widgets/poster-name.js

```diff
@@ -186,7 +188,8 @@ export default createWidget("poster-name", {
         const badgeIcon = new RenderGlimmer(
           this,
           `span.user-badge-button-${badge.slug}`,
-          hbs`<UserBadge @badge={{@data.badge}} @user={{@data.user}} @showName={{false}} />`,
+          hbs`
+            <UserBadge @badge={{@data.badge}} @user={{@data.user}} @showName={{false}} />`,
```

Comment:
There are quite a lot of `hbs` calls which have this weird newline at the beginning. Did something go wrong with prettier?

Reply:
Don't know what happened but running prettier again doesn't change this

Reply:
I tried removing a few of them manually.